l3kernel/l3regex.dtx

```diff
@@ -1167,20 +1167,24 @@
 %   \cs{@@_toks_put_right:Ne} is provided because it is more
 %   efficient than \texttt{e}-expanding with \cs{exp_not:n}.
 %    \begin{macrocode}
-\cs_new_protected:Npn \@@_toks_put_left:Ne #1#2
+\cs_if_exist:NTF \tex_etokspre:D
```

Comment:
This part I like - could we perhaps have a separate PR that only uses these primitives where available? I think that's easier to justify than using more `:D` functions across the board.